easybuild/easyblocks/generic/scons.py

```diff
@@ -54,7 +54,9 @@ def test_step(self):
         Test with SCons 
         """
         cmd = "scons check"
-        (out, _) = run_cmd(cmd, log_all=True)
+        if self.cfg['runtest']:
+            cmd = "scons %s" % (self.cfg['runtest'])
+            (out, _) = run_cmd(cmd, log_all=True)
 
         return out
```

Comment:
@hajgato: +1, I'd just drop the `return` and not catch the return value of `run_cmd`